components/blitz/resources/templates/combined.vm

```diff
@@ -173,7 +173,7 @@ DECLARATION BLOCK:
 #end##ome
 #end##foreach
 
-[$jav] package omero.model;
+[$JAV] package omero.model;
```

Comment:
It includes the line in the `.java` files for abstract classes as well as concrete classes.